backend/app/Http/Controllers/AccountController.php

```diff
@@ -30,4 +31,19 @@ public function show(Account $account)
     {
         return AccountResource::make($account);
     }
+
+    public function store(User $user, AccountPostRequest $request)
+    {
+        $payload = $request->validated();
+
+        $account_type = array_search($payload['account_type'], config('enums.account_type')) + 1;
```

Comment:
better to update the account type to enum (database) to follow the coding standard, it will be easier also to integrate in the backend.